Live/Classes/Item.h

```diff
@@ -0,0 +1,18 @@
+#ifndef __ITEM__H__
+#define __ITEM__H__
+
+#include "GameObject.h"
+
+class Player;
+class Item : public GameObject {
+protected:
+    const unsigned int weight;
```

Comment:
We need to decide on consistent naming on member variables.

Reply:
ok ok, we'll go with m_